### PR TITLE
Template test

### PR DESCRIPTION
**Pull request by Arique1104**
**Contributors:** Arique & Priya

**Details:** This adds sections to our ReadMe and tests our Pull Request Template (which is successful!)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,30 @@
 # Viewing Party
```
Comment (priyapower) [inline README.md:1]: I like that we are working on the ReadMe from the beginning

File: README.md [hunk 2]
```diff
@@ -1,22 +1,30 @@
...
-Viewing party is an application in which users can explore movie options and create a viewing party event for the user and friend's.
+## Features
 
-## Local Setup
+## Code Snippet
 
-1. Fork and Clone the repo
-2. Install gem packages: `bundle install`
-3. Setup the database: `rails db:create`
+### Schema
```
Comment (priyapower) [inline README.md:15]: We are pretty close to being able to upload this right now - however, I agree with you and think we should wait for a finalized schema until the end? Or should we upload the temporary for now?

**Review by priyapower — COMMENTED** [review]
Looks great! Ready to approve this PR

File: README.md (hunk 2 above)
Comment (Arique1104) [reply README.md:15]: I like your schema image that we were reviewing.  I think once we walked through how we would maintain a database where one column was an array was finally ironed out, I felt more comfortable with your idea.  Let's post it and if we change, we can update our image with a before and after photo and simply show it as part of our learning.

File: README.md (hunk 1 above)
Comment (Arique1104) [reply README.md:1]: Me too!

File: README.md (hunk 2 above)
Comment (priyapower) [reply README.md:15]: Sounds good! Can be activity for our standup tomorrow, we can continue working through initial setup and databasing first!

**Pull request merged**